.github/scripts/get-version.js

```diff
@@ -0,0 +1,33 @@
+import { execSync } from 'child_process';
+
+async function getVersion(url) {
+  try {
+    const response = execSync(`curl -s '${url}/graphql' -H 'content-type: application/json' --data-raw '{"query":"{ statistics { appVersion } }"}'`);
```

Comment:
thought (non-blocking): using GQL here is unnecessary, we could just use the statistics endpoint from REST rather than needing to do a post etc.

Reply:
Would simplify the CURL